src/Life/Board.cs

```diff
@@ -23,65 +23,13 @@ public void Evolve()
         throw new NotImplementedException();
     }
 
-    private int NextStateForCell(int i, int j)
```

Comment:
I feel that adding these private methods is more part of the exercise. The public overrides at the bottom aren't used anywhere, so I would say either remove them or include them in tests

Reply:
I agree, if anyone is really struggling we can give this function names as an example to help get them on the right track.